pos_loyalty_redeem_payment/static/src/js/models.esm.js

```diff
@@ -6,7 +6,7 @@ import Registries from "point_of_sale.Registries";
 export const RedeemPaymentOrder = (OriginalOrder) =>
     class extends OriginalOrder {
         has_redeem_payment_lines() {
-            return this.paymentlines.some(
+            return this.pos.config.allow_auto_print_giftcard && this.paymentlines.some(
```

Comment:
Practical config, but IMHO it would be better to leave `has_redeem_payment_lines` with its original behaviour (`true` if any payment line is linked to a voucher) and check `allow_auto_print_giftcard` in `_postPushOrderResolve` and `wait_for_push_order`

Reply:
yes make sense, I'll make the change